### PR TITLE
Changed QaRulesConfiguration to Singleton

### PR DESCRIPTION
**Pull request by chrisjensenuk**
We're retrieving the QaRules from the DB on every page request for Employer and Provider. So suggesting we change to Singleton.

### CODE REVIEW

File: src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -61,7 +61,7 @@ public static void AddRecruitStorageClient(this IServiceCollection services, ICo
                 .Configure<AccountApiConfiguration>(configuration.GetSection("AccountApiConfiguration"))
```
Comment (shomavg) [inline src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs]: Can the namespace be corrected to the location of the file?

Comment (shomavg) [reply src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs]: Also remove unwanted namespace?

Comment (shomavg) [inline src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs]: Change the namespace to point to the correct location.

Comment (shomavg) [reply src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs]: Remove unused namespace please.

Comment (chrisjensenuk) [reply src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs]: 👍

**Pull request merged**